saves/calamity/datapacks/calamity/data/calamity/functions/build_protection/section_small.mcfunction

```diff
@@ -1,19 +1,22 @@
 # Called from: calamity:build_protection/section_medium
 
+# Put the barrier blocks back
+fill ~ 73 ~ ~3 73 ~3 minecraft:barrier replace minecraft:glass
```

Comment:
What is this line doing? It seems like a dev command instead of something still required.

Reply:
Line is supposed to be there. For some reason you cant count barrier blocks with clone filtered so I'm replacing them with glass while counting and then back to barriers.

Changed the comment a little.